### ISSUE TITLE
pfp (profile picture) to username

### ISSUE THREAD

**Issue by StioStudio** [ScratchAddons/ScratchAddons#6270]
### Why this would be helpful

As you know we can not see pfp on pretty much any username on scratch, except on messages, What's Happening? and Explore

### How this addon works

Adding images to the username on: Messages (fav, love and follow), home page rows, profile page rows and studio activity (please say if you know about more)

### Additional context

This is a part of the scratch 4.0 addon
How it could look home page:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/98958235/6b09bcfa-a298-459f-b065-606d2aede455)

**Comment by Samq64** [comment]
Why did you close this and a bunch of your other issues? This one has 3 thumbs ups. If it's because you're not interested in contributing to the project anymore, someone else can still implement your suggestions.

**Comment by StioStudio** [comment]
> Why did you close this and a bunch of your other issues? This one has 3 thumbs ups. If it's because you're not interested in contributing to the project anymore, someone else can still implement your suggestions.

Yes, I am not interested in contributing to the project anymore. Lost my motivation towards scratch addons.

**Comment by BroJac5246** [comment]
> > Why did you close this and a bunch of your other issues? This one has 3 thumbs ups. If it's because you're not interested in contributing to the project anymore, someone else can still implement your suggestions.
> 
> Yes, I am not interested in contributing to the project anymore. Lost my motivation towards scratch addons.

The issues should still stay open, since other people can and will work on them.
